main.py

```diff
@@ -1,7 +1,66 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
-def main() -> QuerySet:
-    pass
+genres = [
+    "Genre",
+    "Action",
+    "Dramma"
+]
+
+actors = {
+    "first_name": [
+        "George",
+        "Kianu",
+        "Scarlett",
+        "Will",
+        "Jaden",
+        "Scarlett"
+    ],
+    "second_name": [
+        "Klooney",
+        "Reaves",
+        "Keegan",
+        "Smith",
+        "Smith",
+        "Johansson"
+    ]
+}
```

Comment:
You could simplify this structure to store a list with tuples that will represent the `first_name` and `second_name`. You could unpack `first_name` and `second_name` during the iteration as well.